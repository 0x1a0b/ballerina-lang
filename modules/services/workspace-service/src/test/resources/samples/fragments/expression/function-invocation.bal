test:function1(a, b)
