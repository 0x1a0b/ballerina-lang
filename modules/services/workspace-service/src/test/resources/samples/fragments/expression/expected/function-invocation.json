{"type":"function_invocation_expression","package_name":"test","package_path":null,"function_name":"function1","line_number":"2","position_info":{"start_line":2,"start_offset":0,"stop_line":2,"stop_offset":0},"whitespace_descriptor":{"regions":{"0":"\n","1":"","2":"","3":""}},"children":[{"line_number":"2","position_info":{"start_line":2,"start_offset":15,"stop_line":2,"stop_offset":15},"whitespace_descriptor":{"regions":{"0":"","3":""}},"type":"simple_variable_reference_expression","variable_reference_name":"a","variable_name":"a","package_name":null},{"line_number":"2","position_info":{"start_line":2,"start_offset":18,"stop_line":2,"stop_offset":18},"whitespace_descriptor":{"regions":{"0":" ","3":""}},"type":"simple_variable_reference_expression","variable_reference_name":"b","variable_name":"b","package_name":null}]}